lib/iris/tests/integration/test_netcdf__loadsaveattrs.py

```diff
@@ -69,6 +69,9 @@ def global_attr(request):
     iris.fileformats.netcdf.saver._CF_DATA_ATTRS
     + iris.fileformats.netcdf.saver._UKMO_DATA_ATTRS
 )
+# Don't test iris_extended_grid_mapping, as it is a special attribute that
+# is not expected to always roundtrip.
+_LOCAL_TEST_ATTRS = [a for a in _LOCAL_TEST_ATTRS if a != "iris_extended_grid_mapping"]
```

Comment:
I really like this precedent of property that is driven by a namespaced attribute, which doesn't get output, and I'd like to see it explored more in the future.

I don't want to hold this PR up any more, but the exceptions that this change needs are _horrible_. Please could you raise an issue for someone to explore special handling of ALL namespaced attributes in the future? The next time someone adds one, they shouldn't need to make any changes to the saving code. Maybe they should all be 'private' too (`_iris_...`)?

Since `iris_extended_grid_mapping` shows where the code is needed, it could probably be marked as `Good First Issue`